From bc24ca932a24518605b3301ac974ac04f9d367e1 Mon Sep 17 00:00:00 2001
From: Luka Logar <luka.logar@cifra.si>
Date: Fri, 29 May 2020 10:43:05 +0200
Subject: [PATCH] maven-antrun-plugin should not depend on tools...

...which under Java 11 don't exist any more

[INFO] --- maven-antrun-plugin:1.8:run (default) @ jcardsim ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.930 s
[INFO] Finished at: 2020-05-29T10:17:24+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (default) on project jcardsim: Execution default of goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run failed: Plugin org.apache.maven.plugins:maven-antrun-plugin:1.8 or one of its dependencies could not be resolved: Could not find artifact com.sun:tools:jar:1.5.0 at specified path /usr/lib/jvm/java-11-openjdk-amd64/../lib/tools.jar -> [Help 1]
---
 pom.xml | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/pom.xml b/pom.xml
index bb4b3da..ee7cac6 100644
--- a/pom.xml
+++ b/pom.xml
@@ -391,13 +391,6 @@
                 <artifactId>maven-antrun-plugin</artifactId>
                 <version>1.8</version>
                 <dependencies>
-                    <dependency>
-                        <groupId>com.sun</groupId>
-                        <artifactId>tools</artifactId>
-                        <version>1.5.0</version>
-                        <scope>system</scope>
-                        <systemPath>${java.home}/../lib/tools.jar</systemPath>
-                    </dependency>
                     <dependency>
                         <groupId>org.apache.ant</groupId>
                         <artifactId>ant-junit</artifactId>
